text/0051-graceful-stack-upgrades.md

```diff
@@ -0,0 +1,41 @@
+# Graceful Stack Upgrades
+
+## Summary
+
+All Paketo Buildpacks should support graceful stack upgrades between the
+current Bionic and Jammy stacks. For most buildpacks, this should not require
```

Comment:
Do you have a concrete example of a buildpack (or at least a type of buildpack) that does not work correctly?

I was initially thinking the problem would be dynamic linking against libraries that have changed, but in those cases we have introduced a new dependency with a recompiled binary dynamically linked against the new stack.

Reply:
I can update the RFC to call out more specifically where I think the issues are likely to occur. I agree that any buildpack that just installs a dependency will likely not have an issue since the dependency should be guaranteed to run on the stack, or be replaced with another version that should.

For buildpacks with possible issues, I think the canonical example would be instances where a package manager installs dependencies that link out to C code provided by the OS. I don't have an example right in front of me, but the worry would be that say `bundle-install` installs a bunch of gems that have C bindings that are then broken by the stack upgrade.

Reply:
RFC updated with more details about the types of buildpacks we will likely need to modify.